### PR TITLE
Event Delegation problem

### PR DESCRIPTION
**Pull request by manmodesanket**
## Description 
Event Delegation using react

## Type
Machine Coding Round

## Related Technology
React, css

## Source
Asked in Ridecell interview

## Working Solution
Codesandbox link: https://codesandbox.io/s/bubbling-problem-8erwc

### CODE REVIEW

File: react-problems/bubbling-problem/public/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
```
Comment (Jaynil1611) [inline react-problems/bubbling-problem/public/index.html:1]: Remove index.html

File: react-problems/bubbling-problem/src/App.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import "./styles.css";
```
Comment (Jaynil1611) [inline react-problems/bubbling-problem/src/App.js:1]: Keep this file & add this file & readme as a folder under
`react-snippets`

File: react-problems/bubbling-problem/src/index.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { StrictMode } from "react";
```
Comment (Jaynil1611) [inline react-problems/bubbling-problem/src/index.js:1]: Remove this file

File: react-problems/bubbling-problem/src/styles.css [hunk 4]
```diff
@@ -0,0 +1,34 @@
+.App {
```
Comment (Jaynil1611) [inline react-problems/bubbling-problem/src/styles.css:1]: Remove this file

File: react-problems/bubbling-problem/README.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+## Question Description
+
+Render the movie data similar to following:
+
+![Table](https://raw.githubusercontent.com/manmodesanket/content-archives/main/screenshot.png)
+
+After clicking on each cell an alert dialog should popup with respective movie data.
+
+If user clicks on "Imdb Ratings" then an alert dialog should popup with movie rating.
+
```
Comment (Jaynil1611) [inline react-problems/bubbling-problem/README.md:10]: Add ##Solution as a heading & then add the link
Change the title from `CodeSandbox Link` to `<question-name> Solution` & add the link under the whole title

**Review by Jaynil1611 — CHANGES_REQUESTED** [review]
The bubbling problem name seems very vague
Give a more descriptive problem name

**Comment by manmodesanket** [comment]
Made changes as per the request.

**Review by Jaynil1611 — APPROVED** [review]
@manmodesanket Change the sandbox name to `Event Delegation` as well

**Pull request merged**